docs/scripting.md

```diff
@@ -566,27 +566,40 @@ Scripts loaded from the `scripts/` directory are loaded in alphabetical order, s
 
 Once you've built some new scripts to extend the abilities of your robot friend, you should consider sharing them with the world! At the minimum, you need to package up your script and submit it to the [Node.js Package Registry](http://npmjs.org). You should also review the best practices for sharing scripts below.
 
+## See if a script already exists
+
+Check out the [NPM Registry](https://www.npmjs.com/browse/keyword/hubot-scripts) for the `hubot-scripts` keyword.  If you don't see an existing package that you can contribute to, then you can easily get started using the `hubot` script [yeoman](http://yeoman.io/) generator.
+
```

Comment:
What do you think about linking to https://hubot.github.com/docs/#scripts instead of duplicating that info here? That way, if we change how to find scripts (such as building a script directory: https://github.com/github/hubot/issues/976), we only have to update the docs in one place.